org.unitystation.StationHub.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: org.unitystation.StationHub
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: stationhub.sh
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --filesystem=~/.var/app/org.unitystation.StationHub:create
+  - --talk-name=org.freedesktop.Flatpak  
```

Comment:
What is it needed for?

Reply:
- --device=all : controllers
- --share=network : it's an online game
- --filesystem=... : it creates new folders as part of managing the installs
- --talk-name=org.freedesktop.Flatpak : I had to make a flatpak specific patch because upstream dotnet's ping function does not work in sandboxes at all. It's only used in ServerWrapper.cs to run /usr/bin/ping on the user's system because lots of distros' default SELinux permissions will not let that work anywhere else.

Reply:
`--talk-name=org.freedesktop.Flatpak` pretty much throws any sandboxing away. I played a bit in the Sdk shell and it's possible to build iputils to have `ping` available without using `flatpak-spawn` (with `-DBUILD_ARPING=false -DBUILD_CLOCKDIFF=false -DBUILD_NINFOD=false -DBUILD_RDISC=false -DBUILD_TRACEPATH=false -DENABLE_RDISC_SERVER=false` passed to meson). I tested it on Fedora with SELinux enabled and I could ping arbitrary hostnames/IPs.